app/i18n/nl/gen.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/* Dutch translation by Wanabo. http://www.nieuwskop.be */
+return array(
+	'action' => array(
+		'actualize' => 'Actualiseren',
+		'back_to_rss_feeds' => '← Ga terug naar je RSS feeds',
+		'cancel' => 'Annuleren',
+		'create' => 'Opslaan',
+		'disable' => 'Uitzetten',
+		'empty' => 'Leeg',
+		'enable' => 'Aanzetten',
+		'export' => 'Exporteren',
+		'filter' => 'Filteren',
+		'import' => 'Importeren',
+		'manage' => 'Beheren',
+		'mark_read' => 'Markeer als gelezen',
+		'mark_favorite' => 'Markeer als favoriet',
+		'remove' => 'Verwijder',
+		'see_website' => 'Bekijk website',
+		'submit' => 'Opslaan',
+		'truncate' => 'Verwijder alle artikelen',
+	),
+	'auth' => array(
```

Comment:
Thanks for the feedback. I was referring to the `auth` section:

* en: https://github.com/Alkarex/FreshRSS/blob/Dutch/app/i18n/en/gen.php#L23
* nl: https://github.com/Alkarex/FreshRSS/blob/Dutch/app/i18n/nl/gen.php#L23